lib/kumascript/firelogger.js

```diff
@@ -42,7 +42,7 @@ module.exports = function (options) {
 
             // Patch the Vary: header to include X-FireLogger, to indicate that
             // the response changes based on its value.
-            var curr_vary = '' + res.get('Vary'),
+            var curr_vary = res.get('Vary') || '',
```

Comment:
This was a bug: if there was no Vary header, this line was evaluating to "undefined" which made a test fail